OutOfSchool/OutOfSchool.IdentityServer/Resources/SharedResource.en.resx

```diff
@@ -300,10 +300,31 @@
   <data name="Email {0} is already taken" xml:space="preserve">
     <value>Email {0} is already taken</value>
   </data>
-   <data name="Not registered yet?" xml:space="preserve">
+  <data name="Not registered yet?" xml:space="preserve">
     <value>Not registered yet?</value>
   </data>
   <data name="Save" xml:space="preserve">
     <value>Save</value>
   </data>
+  <data name="Change email {0} sent" xml:space="preserve">
+    <value>Instruction to change email have been sent to {0}. Please check the email and follow the steps in the letter.</value>
```

Comment:
Please fix grammar in `Instruction to change email have been sent to` - `... has been sent ...` or maybe even simple `... is sent ...`

Reply:
(sorry that didn't catch this last time)